akd/src/storage/types.rs

```diff
@@ -38,6 +40,10 @@ pub struct Values(pub String);
 #[derive(Debug, Clone, Hash, Eq, PartialEq, Serialize, Deserialize)]
 pub struct ValueStateKey(pub String, pub u64);
 
+/// Root hash of the tree and its associated epoch
```

Comment:
This is no longer used right? Can you clean revert this file, then it should be good to merge!